addons/website_sale/views/templates.xml

```diff
@@ -2034,6 +2034,9 @@
                                                 </div>
                                             </div>
                                         </t>
+                                        <t t-else="">
+                                            <input type="hidden" class="form-check-input mr8" name="use_same" t-att-value="partner_id == website_sale_order.partner_shipping_id.id == website_sale_order.partner_invoice_id.id"/>                                       
+                                        </t>
```

Comment:
Is the `class` attribute necessary ?
I suppose it doesn't change anything since it's a hidden input :thinking:

Reply:
You are right , copy paste mistake